### PR TITLE
feat(classification): Integrating Security Controls with Classification

### PR DESCRIPTION
**Pull request by jpfranco**
Just a quick update to bundle the `SecurityControls` with the `Classification` component since they will be used together most of the time.

Also renamed the main prop for `SecurityControls` to `controls` (instead of `accessPolicyRestrictions` in order to keep it more generic and allow future use cases.

Screen Shot 2019-11-19 at 1 12 48 PM

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jpfranco has signed the CLA. Thanks for the pull request!

File: src/features/classification/security-controls/SecurityControls.js [hunk 1]
```diff
@@ -14,24 +14,24 @@ import './SecurityControls.scss';
 const { FULL, SHORT, SHORT_WITH_TOOLTIP } = SECURITY_CONTROLS_FORMAT;
 
 type Props = {
-    accessPolicyRestrictions: AccessPolicyRestrictions,
+    controls: AccessPolicyRestrictions,
```
Comment (priyajeet) [inline src/features/classification/security-controls/SecurityControls.js]: should type also be renamed?

Comment (jpfranco) [reply src/features/classification/security-controls/SecurityControls.js]: Yes, updated

File: src/features/classification/Classification.js [hunk 2]
```diff
@@ -44,6 +61,9 @@ const Classification = ({ definition, className = '', messageStyle, name, onClic
                     <FormattedMessage {...messages.missing} />
                 </span>
             )}
+            {controls && isSecurityControlsEnabled && (
```
Comment (jpfranco) [inline src/features/classification/Classification.js]: Checking both vars mostly for flow's sake

**Pull request merged**